web/fluidity.money/src/components/RewardsBackground/RewardsBackground.tsx

```diff
@@ -23,14 +23,27 @@ interface Reward {
   transaction: string;
 }
 
-const rewardLimit = 50
+interface CarouselInfo {
+  startIndex: number;
+  endIndex: number;
+}
 
 const RewardsBackground = () => {
   const { apiState, chain, network } = useChainContext();
   const { ref, inView } = useInView();
   const { width } = useViewport();
   const { weekWinnings } = apiState;
 
+  let iteratorCounter = 0;
```

Comment:
More of a functional programming thing, but you can get rid of this using the index inside of the map down below

Reply:
uhm, yeah, i needed to handle the incremental operation of it myself, instead of letting js internals do it for me, cos of the way i use it, suss line 125 and 135 in `RewardsBackground.tsx` incrementing 4 times on a single iteration. if you get what i mean.